### PR TITLE
Actions documentation

### PR DESCRIPTION
**Pull request by dpgraham**
(also fixed a typo in flick/)

### CODE REVIEW

**Comment by dpgraham** [comment]
(related to: https://github.com/appium/appium/issues/11184)

File: commands-yml/commands/interactions/actions.yml [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+      description: The type of action. For any input source it can be 'pause'. For 'pointer' input source 'pointerMove', 'pointerUp' or 'pointerDown'. For 'key' it can be 'keyDown' or 'keyUp'
+    - name: actions[$INDEX].actions[$INDEX].value
+      type: string
+      description: For a 'keyUp' or 'keyDown' action thevalue to send to the keyboard. Should be a one-character string ("s", "\uE009")
+    - name: actions[$INDEX].actions[$INDEX].duration
+      type: number
+      description: How long to perform the action in 'ms'. Only applicable to 'pause' and 'pointerMove'.
+    - name: actions[$INDEX].actions[$INDEX].origin
+      type: string|object
+      description: For 'pointerMove', this tells the input source what x,y are relative to. Can be 'viewport', 'pointer' or \{'element-6066-11e4-a52e-4f735466cecf': '<ELEMENT_ID>'\}
```
Comment (imurchie) [inline commands-yml/commands/interactions/actions.yml]: I think you need `&#58;` instead of `:`. Colons are not allowed in yaml.

**Comment by mykola-mokhnach** [comment]
I'd also add some example about how to type keys. BTW, this is not supported by XCUITest driver.

**Comment by dpgraham** [comment]
Okay... yeah typing examples would be helpful, especially for the Java client.

**Comment by dpgraham** [comment]
Fixes pushed

File: commands-yml/commands/interactions/actions.yml [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+  javascript_wdio: "http://webdriver.io/api/protocol/actions.html"
+  #javascript_wd: "https://github.com/admc/wd/blob/master/lib/commands.js#L44"
+  ruby: "https://www.rubydoc.info/gems/selenium-webdriver/Selenium/WebDriver/W3CActionBuilder"
+  php: "https://github.com/jupeter/selenium-php-webdriver/blob/master/Webdriver/Common/ActionChains.php"
+  csharp: "https://github.com/appium/appium-dotnet-driver/" # TODO
+
+# Driver support by platform
+driver_support:
+  ios:
+    xcuitest: false
```
Comment (mykola-mokhnach) [inline commands-yml/commands/interactions/actions.yml]: xcuitest only supports pointers of type touch and nothing else

Comment (dpgraham) [reply commands-yml/commands/interactions/actions.yml]: @mykola-mokhnach I updated it to `true`... when we do a code sample we can make a note about it not support key events on iOS.

**Pull request merged**